WordPress/WordPressTest/TitleSubtitleHeaderTests.swift

```diff
@@ -0,0 +1,55 @@
+
+import XCTest
+@testable import WordPress
+
+final class TitleSubtitleHeaderTests: XCTestCase {
+    private var header: TitleSubtitleHeader?
+
+    override func setUp() {
+        super.setUp()
+        header = TitleSubtitleHeader(frame: .zero)
+    }
+
+    override func tearDown() {
+        header = nil
+        super.tearDown()
+    }
+
+    func testHeaderContainsTitleLabel() {
+        XCTAssertNotNil(header?.titleLabel())
+    }
+
+    func testHeaderContainsSubtitleLabel() {
+        XCTAssertNotNil(header?.subtitleLabel())
+    }
+
+    func testSetTitleSetsTitleLabel() {
+        let title = "🦄"
+        header?.setTitle(title)
+
+        XCTAssertEqual(header?.titleLabel().text, title)
+    }
+
+    func testSetSubtitleSetsSubtitleLabel() {
+        let text = "🦄🦄"
```

Comment:
As much as I love this emoji, are there any complications that could occur from us using these in tests? If not, I'm all for using them.

Reply:
I've been doing emojis in unit tests since Swift was announced. I think I recall reading somewhere in [the first edition of this](https://itunes.apple.com/us/book/swift-programming-language/id881256329?mt=11) that there was full unicode support. I'll remove them anyway